docs/reference/migration/migrate_6_3.asciidoc

```diff
@@ -24,6 +24,24 @@ Similarly, the `suggest` index metric on the `indices` metric on the nodes stats
 API has provided a response containing only an empty object since 5.0.0. In
 6.3.0 this metric has been deprecated.
 
+[[breaking_63_packaging_changes]]
+=== Packaging changes
+
+Starting with version 6.3, all of the {xpack} features ship with the default 
+distribution of {es}. You no longer need to install {xpack} separately. 
+
+==== Removal of the `x-pack` configuration directory
```

Comment:
After digging into this, we've found that the breaking change is a bug. So maybe this should be a known issue instead? The fix is in https://github.com/elastic/elasticsearch/pull/33688. The issue is the following:

* Pre 6.3 with x-pack installed put the default configuration files for security in ES_PATH_CONF/x-pack
* 6.3+ puts the default configuration files in ES_PATH_CONF/x-pack
* A BWC layer was added to look in the `ES_PATH_CONF/x-pack` *if* the file did not exist in `ES_PATH_CONF/x-pack`. This essentially means that the old files are ignored since the new default files are there. The workaround is to move the old files up a level (out of the x-pack directory)

Reply:
No it will not be backported to 6.3